src/python/WMCore/BossAir/PyCondorAPI.py

```diff
@@ -0,0 +1,12 @@
+import htcondor
+
+def isJobLimitReachedInSchedd():
```

Comment:
I'd name it `isScheddOverloaded`, since it will not rely only on the job limit but on the duty cycle and *anything* else that the glideinWMS team decides to configure for this `CurbMatchmaking` ad.